pkg/agent/plugin/svidstore/disk/xattr_fallback.go

```diff
@@ -0,0 +1,15 @@
+// +build !linux
+// +build !darwin
```

Comment:
what do you think about 1 file for windows and another for !windows?
we are using that approach in another places, and may works

Reply:
The problem is that there are non-Windows platforms where this is not supported, so we can't just have !windows for the supported implementation. The supported ones are: linux, darwin, freebsd and netbsd. I see that I missed freebsd and netbsd., which I should add.